handbook/sales/index.md

```diff
@@ -20,9 +20,34 @@ The Sales team represents us and our values to customers, bringing back dollars
 
 ## Definitions
 
-### ARR
+## [Monthly Active User (MAU)](https://docs.sourcegraph.com/admin/subscriptions#how-active-users-are-counted)
 
-Annual Recurring Revenue (ARR) is the dollar value of contracted recurring revenue in a (normalized) one-year period.
+### Annual Recurring Revenue (ARR)
+
+Annual Recurring Revenue (ARR) is the dollar value of contracted recurring revenue for a one-year period. ARR is MRR times 12. 
+
+### Monthly Recurring Revenue (MRR)
+
+Monthly Recurring Revenue (MRR) is the dollar value of contracted recurring revenue for a one-month period, and excludes contracted work, paid training sessions, and other sources of one-time revenue. The calculations are:
+- For a fixed user contract, we go under the assumption of a renewal: Total Contract Value/# of months in contract.
+- For the MAU pricing model: Rate times number of active users during the month.
+
+### Total Monthly Active Paying Users
+
+Total Monthly Active Paying Users is a sum of all MAUs from all customers.   
+
+### Dollar-Based Net Expansion Rate
+
+Dollar-based Net Expansion Rate is the rate at which the revenue from a customer or customer cohort grows from time period to the next. For example, if we wanted to calculate the net expansion rate from 2018 to 2019, the calculation would be 2019-12 MRR/2018-12 MRR. 
+
+### Magic Number
+
+The Magic Number is the incremental (new) recurring revenue for trailing three months / sales and marketing between the trailing 4 and 6 months. 
```

Comment:
Interesting ROI metric, but I haven't actually seen this in use internally — is it something you're proposing, or ad addition from Adam?

Reply:
Unless we actually use this (or will immediately start using it), please remove this because
[The handbook documents what we do right now. It doesn’t describe the ideal practices we hope to follow in the future.](https://about.sourcegraph.com/handbook/usage#handbook-guidelines)